apps/bridge/src/components/DeprecationModal/DeprecationModal.tsx

```diff
@@ -0,0 +1,44 @@
+import { useCallback, useMemo } from 'react';
+import { Modal } from 'apps/bridge/src/components/Modal/Modal';
+import { useLocalStorage } from 'usehooks-ts';
+
+export function DeprecationModal() {
+  const [deprecationSeen, setDeprecationSeen] = useLocalStorage('deprecationSeen', false, {
+    // make SSR compatible
+    initializeWithValue: false,
+  });
+
+  const onClose = useCallback(() => {
+    setDeprecationSeen(true);
+  }, [setDeprecationSeen]);
+
+  const content = useMemo(() => {
+    return (
+      <div className="flex flex-col items-center space-y-8">
+        <p>
+          Starting next week, users will be redirected to the Superchain Bridges. See FAQs for
```

Comment:
Let me check on adding the updated copy with links, here (and also accompanying FAQs)

Reply:
Sounds good.